src/_container.scss

```diff
@@ -0,0 +1,45 @@
+.container {
+  min-height: 645px;
+  width: auto;
+  margin: 0;
+  padding: 0;
+  background-color: $color--darker;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: 104px 4fr 3fr 4fr;
+  align-content: unset;
+  justify-content: stretch;
+  grid-gap: 5px;
+  grid-template-areas: 
+  "headline headline headline headline"
+  "buttonbox buttonbox buttonbox buttonbox"
+  "datebox datebox datebox datebox"
+  "confbox confbox modelbox modelbox"; 
+
+  &__box {
```

Comment:
These styles should be moved to the `.card` component.

Reply:
Unfortunately, I don't understand what do You mean.
.container need to be rename to . card????

Reply:
"Card" is a standalone UI component, so it can be considered a _block_.
In other words, what you have declared as an _element_ (.container__box) is actually a _block_.

What I meant is that you should rename `.container__box` to `.card` (and move it to a separate file).